src/main/resources/labels/marcRel.json

```diff
@@ -737,9 +777,9 @@
 	},
 	{
 		"uri": "http://id.loc.gov/vocabulary/relators/pat",
-		"label": "Auftraggeber/in",
+		"label": "Im Auftrag von",
 		"multilangLabel": {
-			"de": "Auftraggeber/in",
+			"de": "Im Auftrag von",
 			"en": "commissioning body"
```

Comment:
You begin German with uppercase but English is lower cased. This should be harmonized.

Reply:
i follow the DNB document here: file:///tmp/mozilla_tobias0/AH-017.pdf

Also I think stand alone nones in english are written in lower case. e.g. in dictionaries

Reply:
on the other hand: https://www.loc.gov/marc/relators/relacode.html here it is in upper case

Reply:
you got a point, I don't know. You may leave it as it is, then.
Wait:
The labels are supposed to be used as a fallback to name a fields in a catalog. Looking at amazon.com and others it seems to be rather usual to use uppercase to do this.